libs/core/src/lib/feed-list-item/components/item/feed-list-item.component.scss

```diff
@@ -0,0 +1,51 @@
+$block: fd-feed-list;
+
+.#{$block} {
```

Comment:
Removed unnecessary. But related to other - I improved some cases in dynamic elements.
- `__item--collapsible` needed for correct working line-clamp 
- `__name a` needed if we have very long author name
- `__thumb` predefined place for your avatar, for more predictive behavior
- `__link--more` unfortunately, but link have no cursor pointer without `href` attribute. I think, it would be great that `fd-link` has that by default.

Reply:
Thanks for clarification :)